examples/app/export_csv/main.py

```diff
@@ -19,7 +19,7 @@ def update():
         'years_experience' : current.years_experience,
     }
 
-slider = Slider(title="Max Salary", start=10000, end=250000, value=150000, step=1000)
+slider = Slider(title="Max Salary", start=10000, end=110000, value=50000, step=1000, format="0,0")
```

Comment:
Don't we have a working double slider now? It might be nice to update to that for a salary range